charts/pega/README.md

```diff
@@ -130,6 +130,30 @@ docker:
     imagePullPolicy: "Always"
 ```
 
+## Environment Name (Optional)
+
+Set the environment name to help differentiate this deployment from others.  The provided name will be used as a prefix to the various Pega tiers (and the k8s objects created on their behalf).  The environment name must consist of a DNS friendly name as it will be incorporated into the names of k8s objects. For example:
+```yaml
+global:
+  environment:
+    name: app1-dev-
```

Comment:
Is this tied into the global.domain parameter that is used for the hostname for this environment?  It seems like it would be less confusing if the environment name value were derived from that value.

Reply:
No.

I am not familiar with global.domain (only global.tier[].ingress.domain).  Can you point me at a reference?

Reply:
Sorry - I meant global.dnsname.  I believe this is used for configuration for the ingress.
That said, I also noticed that there is a global.ingress.domain param.  Can all of these be consolidated?  It seems like they all could potentially be set to the same value.

Reply:
I couldn't find a reference to 'dnsname' grepping through the project.

I do see a couple of 'domain' args:

- global.tier[].ingress.domain 
- global.tier[].service.domain
- global.tier[].domain

Though I'm inclined to say that we should not derive this from the domain used for ingress.  My thought is that these names would be internal identifiers that don't necessarily relate to their public domain names.  My intention in bringing up DNS naming conventions was to indicate the constraints on the names given:

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names

(Don't want folks thinking they can include spaces and such.)

Reply:
global.dnsname is referenced in the sce.  @zitikay do you have any thoughts about this?

Reply:
Actually, @misterdorito you added the dnsname param (in the SCE).  I'm just trying to avoid the confusion of having similar parameters with slightly different uses.  

How would you differentiate the following to a user?
global.tier[].ingress.domain
global.dnsname
global.environment.name

Reply:
global.dnsname: This is only used in the SCE and is largely need to work around the limited utility of the SCE tooling.  It is internal to the SCE and among the things that I'd like to refactor out of existence -- as will probably happen soon given my understand of work to come in the near future (as there is need for multiple ingresses for different  types of deployments.

I'd rather we did not derive these 'deployment.name'-s (see other comments below regarding the change of nomenclature) from ingress/dns names as I'd expect that these will be relatively short names use to distinguish deployments (mostly so that PDC is capable of reporting on stream nodes in client-managed cloud scenarios).  I'd expect this not to be widely used if PDC is not a concern.  For SCE deployments, this will not be used.  And as there are no guarantees that multiple ingresses in the same deployment will have commonality in their domains.